iocore/net/quic/QUICLossDetector.cc

```diff
@@ -562,7 +501,6 @@ QUICLossDetector::_retransmit_lost_packet(QUICPacketUPtr &packet)
 
     reactor->on_frame_lost(frame_info.id());
   }
-  this->_transmitter->retransmit_packet(*packet);
```

Comment:
Remove `_transmitter`  Because we don't need this.

Reply:
Can you put a Debug log instead, like `QUICNetVConnection::retransmit_packet` does?

Reply:
Will do